modules/datadict/README.md

```diff
@@ -0,0 +1,43 @@
+# Data Dictionary
+
+## Purpose
+
+The data dictionary module is used for viewing and modifying the
+LORIS data dictionary.
+
+## Scope
+
+The module displays and manages the data dictionary for things
+already stored in the LORIS `parameter_type` table. This table is
+generally autopopulated and provides a dictionary for imaging headers
+(populated by the imaging pipeline) and behavioural instrument
+(populated by the `lorisform_parser.php` script).
+
+It does not provide a way to enter new things into the data dictionary,
+or describe the data dictionary for arbitrary SQL tables which are
+not stored in the `parameter_type` table, such as instrument flags
+or candidate/session data.
+
+## Permission
+
+The `data_dict_view` permission allows the user to view the LORIS
+data dictionary.
+
+The `data_dict_edit` permission allows the user to both view, and
+modify the description of, rows in the data dictionary.
+
+## Configuration
+
+None
```

Comment:
(Per what was discussed in LORIS meetings, any steps required to set it up should be in this Configuration section:)
* To populate the Data Dictionary with instrument fields, users must run a script in the tools/ directory: `lorisform_parser.php`
As described above, imaging fields are populated in the data dictionary when the imaging pipeline inserts scans into LORIS.